validation/connector_plugin_manifest_latest.xsd

```diff
@@ -94,9 +94,40 @@
       <xs:pattern value="(\d+)((\.)(\d+))*((-)(\d+))*"/>
     </xs:restriction>
   </xs:simpleType>
+  <xs:group name="VendorInformation-G">
+    <xs:sequence>
+      <xs:element minOccurs="0" name="vendor-information">
+        <xs:complexType>
+          <xs:sequence>
+            <xs:group ref="Company-G"/>
+            <xs:group ref="SupportLink-G"/>
+          </xs:sequence>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:group>
+  <xs:group name="Company-G">
+    <xs:sequence>
+      <xs:element name="company">
+        <xs:complexType>
+          <xs:attribute name="name" type="xs:string" use="required"/>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:group>
+  <xs:group name="SupportLink-G">
+    <xs:sequence>
+      <xs:element name="support-link">
+        <xs:complexType>
+          <xs:attribute name="url" type="xs:anyURI" use="required"/>
+        </xs:complexType>
+      </xs:element>
+    </xs:sequence>
+  </xs:group>
   <xs:element name="connector-plugin">
     <xs:complexType>
       <xs:sequence>
+        <xs:group ref="VendorInformation-G"/>
```

Comment:
So we're making this element required in the SDK?  The postgres_odbc/jdbc samples will need to be updated also if so.  How is this made optional on the platform side?

Reply:
No, it is still optional here. See line 99 `<xs:element minOccurs="0" name="vendor-information">`. 
XSD after changes should be same as monolith's xsd.